### PR TITLE
Remove NextContent, use Footer on all pages

### LINKED ISSUE #586: Details Page - Replace "Next Content" footer
In Carte, the colored next content footer has been removed and replaced with standard site footer.

Screen Shot 2020-03-26 at 2 33 40 PM

### PR DESCRIPTION
**Pull request by taysea**
NextContent doesn't persist into Carte designs, now Footer is used across all pages. Any tests for `NextContent` have been removed.

Closes #586

### CODE REVIEW

**Comment by britt6612** [comment]
were the tests not failing when you took this out ?

**Comment by britt6612** [comment]
Im going to link this to issue #562 it was part of taking out the tests

**Comment by britt6612** [comment]
Code wise looks good! It will be good once we get related or something else on pages now it feels off having to go back to get to another section. 👍

File: aries-site/src/layouts/main/Layout.js [hunk 1]
```diff
@@ -69,15 +63,10 @@ export const Layout = ({
                 }}
               >
                 {children}
-                {!isLanding && !isNavPage && (
-                  <>
-                    <FeedbackSection />
-                    <NextContent color={color} nextContent={nextContent} />
-                  </>
-                )}
+                {!isLanding && !isNavPage && <FeedbackSection />}
```
Comment (halocline) [inline aries-site/src/layouts/main/Layout.js]: Is `isNavPage` used any where any more? If not, can we remove it from this file altogether?

File: aries-site/src/layouts/main/Layout.js [hunk 2]
```diff
@@ -69,15 +63,10 @@ export const Layout = ({
                 }}
```
Comment (halocline) [inline aries-site/src/layouts/main/Layout.js]: Can the conditional on line 61 be removed? I think this should now be consistent across all pages.

File: aries-site/src/layouts/main/Layout.js (hunk 1 above)
Comment (taysea) [reply aries-site/src/layouts/main/Layout.js]: You're right, we're good to remove this now.

File: aries-site/src/layouts/main/Layout.js (hunk 2 above)
Comment (taysea) [reply aries-site/src/layouts/main/Layout.js]: Yep! Good call.

**Pull request merged**